File: TransCoder_tokenized_test_set_functions/java/FREQUENT_ELEMENT_ARRAY_1.java
static int mostFrequent ( int arr [ ], int n ) {
  Map < Integer, Integer > hp = new HashMap < Integer, Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    int key = arr [ i ];
    if ( hp . containsKey ( key ) ) {
      int freq = hp . get ( key );
      freq ++;
      hp . put ( key, freq );
    }
    else {
      hp . put ( key, 1 );
    }
  }
  int max_count = 0, res = - 1;
  for ( Entry < Integer, Integer > val : hp . entrySet ( ) ) {
    if ( max_count < val . getValue ( ) ) {
      res = val . getKey ( );
      max_count = val . getValue ( );
    }
  }
  return res;
}

File: results/java/FREQUENT_ELEMENT_ARRAY_1.cpp
int f_filled ( int arr [ ] , int n ) {
  map < int , int > hp ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    int key = arr [ i ] ;
    if ( hp . find ( key ) != hp . end ( ) ) {
      int freq = hp [ key ] ;
      freq ++ ;
      hp [ key ] = freq ;
    }
    else hp [ key ] = 1 ;
  }
  int maxCount = 0 , res = - 1 ;
  for ( map < int , int > :: iterator val = hp . begin ( ) ;
  val != hp . end ( ) ;
  val ++ ) {
    if ( maxCount < val -> second ) {
      res = val -> first ;
      maxCount = val -> second ;
    }
  }
  return res ;
}
